source/authentication/email-password.txt

```diff
@@ -20,6 +20,20 @@ Overview
 Email/password authentication lets users register and login using an email address.
 Atlas App Services must confirm Email/Password users before they may log in.
 
+Users may choose one of three ways to handle user confirmation:
+
+- :ref:`Send a Confirmation Email <auth-send-a-confirmation-email>`
+
+- :ref:`Run a Confirmation Function <auth-run-a-confirmation-function>`
+
+- :ref:`Automatically Confirm Users <auth-automatically-confirm-users>`
+
+The following diagram shows the process flow for logging in:
+
+.. figure:: /images/email-password-login-flow.png
```

Comment:
Love this diagram! But now I want relevant diagrams for _all_ the authentication providers. 🤔